build-info-extractor-docker/src/main/java/org/jfrog/build/extractor/docker/DockerAgentUtils.java

```diff
@@ -0,0 +1,142 @@
+package org.jfrog.build.extractor.docker;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.BuildImageCmd;
+import com.github.dockerjava.api.model.AuthConfig;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
+import com.github.dockerjava.core.DockerClientBuilder;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.core.command.BuildImageResultCallback;
+import com.github.dockerjava.core.command.PullImageResultCallback;
+import com.github.dockerjava.core.command.PushImageResultCallback;
+import com.github.dockerjava.netty.NettyDockerCmdExecFactory;
+import org.apache.commons.lang.StringUtils;
+import org.jfrog.build.api.util.Log;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Map;
```

Comment:
This import seems to be redundant now.

Reply:
it is done automatically by IntelliJ formatting.

Reply:
You should have removed this import after stopping to use the Map.